app/connectors/exchange/FastPassDetails.scala

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2016 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package connectors.exchange
+
+import forms.FastPassForm._
+import play.api.libs.json.Json
+
+case class FastPassDetails(applicable:Boolean,
```

Comment:
I would add Exchange as a suffix to FastPassDetails. I am not sure if this is the convention but I have seen it in several places.